src/ui/contexts/InstantiateContext.tsx

```diff
@@ -18,7 +19,7 @@ import {
 } from 'types';
 import { useStepper } from 'ui/hooks/useStepper';
 import { useDatabase } from 'ui/contexts/DatabaseContext';
-import { onInsantiateFromHash, onInstantiateFromCode, createInstantiateTx, NOOP } from 'api';
+import { onInsantiateFromHash, onInstantiateFromCode, createInstantiateTx, NOOP, transformUserInput } from 'api';
```

Comment:
Can we give the function `transformUserInput` a more meaningful name? And/or give its return value a type? Otherwise it remains unclear to what this function transforms the `userInput`.